endpoints/appr/cnr_backend.py

```diff
@@ -12,7 +17,97 @@
 from util.request import get_request_ip
 
 
+class LazyPackage(object):
```

Comment:
Can you add a comment saying what this is and why it will be helpful?

Reply:
Sure

Reply:
Why not make this a doc string?

Reply:
Because I don't want this to be treated as a "defined" class for use otherwise. Its merely a workaround